src/Generator/Generators/HtmlOmdGenerator.cs

```diff
@@ -252,7 +251,7 @@ private string GetIcon(ISymbol type, string content)
                 icon += "event";
             if(type.IsStatic && type.ContainingType?.TypeKind != TypeKind.Enum)
             {
-                content = "<span class='static'/>" + content;
+                content = "<span class='static'></span>" + content;

```

Comment:
I don't get the point of this.

Reply:
Did you have static members? A self-closing span isn't valid html